docker/test-server/Dockerfile

```diff
@@ -21,18 +21,19 @@ RUN set -ex ;\
     cd /app ;\
     mkdir -p /etc/etebase-server ;\
     cp docker/test-server/etebase-server.ini /etc/etebase-server ;\
+    cp docker/test-server/etebase-server.ini /app ;\
     sed -e '/ETEBASE_CREATE_USER_FUNC/ s/^#*/#/' -i /app/etebase_server/settings.py ;\
     chmod +x docker/test-server/entrypoint.sh
 
 # this is a test image and should start up quickly, so it starts with the DB
 # and static data already fully set up.
 RUN set -ex ;\
-    cd /app ;\
-    python manage.py migrate ;\
-    python manage.py collectstatic --noinput
+    cd /app # ;\
```

Comment:
Can you explain this change?

Reply:
I have created a new file **entrypoint.sh**, and moved these lines to there, as while using docker-compose, we should execute these lines only when container starts.